src/phoenix/trace/trace_dataset.py

```diff
@@ -37,6 +54,38 @@ def normalize_dataframe(dataframe: DataFrame) -> "DataFrame":
     return dataframe
 
 
+def _delete_empty_document_metadata(documents: Any) -> Any:
+    """
+    Adds a dummy field to the document metadata to make the object serializable over parquet
+    """
+    # If the documents is a list, iterate over them, check that the metadata is
+    # a dict, see if it is empty, and if it's empty, delete the metadata
+    if isinstance(documents, list):
+        # Make a deep copy of the documents
+        documents = copy.deepcopy(documents)
+        for document in documents:
+            metadata = document.get(DOCUMENT_METADATA)
+            if isinstance(metadata, dict) and not metadata:
+                # Delete the metadata object since empty dicts are not serializable
+                del document[DOCUMENT_METADATA]
+    return documents
+
+
+def get_serializable_spans_dataframe(dataframe: DataFrame) -> DataFrame:
+    """
+    Returns a dataframe that can be serialized to parquet. This means that
+    the dataframe must not contain any unserializable objects. This function
+    will delete any unserializable objects from the dataframe.
+    """
+    # Check if the dataframe has any document columns
+    for document_column in DOCUMENT_COLUMNS:
+        if document_column in dataframe.columns:
+            dataframe[document_column] = dataframe[document_column].apply(
+                _delete_empty_document_metadata
+            )
```

Comment:
Here's a fancier version (you can also see why ppl likes `df`: it just makes everything shorter 😊)
```suggestion
    for name, column in df[df.columns.isin(DOCUMENT_COLUMNS)].items():
        df[name] = column.apply(_delete_empty_document_metadata)
```